### ISSUE TITLE
Create the services dashboard

### ISSUE THREAD

**Issue by noobling** [noobling/Homeforenow#11]
This seems rather difficult to implement at this stage.

Some ideas

* List of their services
* Statistics and graphs
* Add service feature
* Edit details of themselves (service provider)

**Comment by thaingu** [comment]
![dashboard](https://user-images.githubusercontent.com/34293984/34409454-deb4f6e6-ec04-11e7-8608-ec7c9bcc2fe5.png)

So what I have brainstormed so far is this.
The page will consist of a bar and panels. This is due to the grid system of bootstrap, using panels that match up with the grids will allow the page to be responsive very easily as the panels can simply just move downwards and stack on each other.
A. will be a bar that has a summary of all the important information for the service, such as name, phone number, location, how much beds they can hold, genders and whether a vacancy is available. This bar will be the first thing a user will see and as such I want a user to be able to view all the vital information from a quick glance. Vacancies can be updated from this as well, with a modal that allows the updating to be quick.  Details of themselves can be edited her as well.
B.  will be a sorta history panel. It will detail the last few updates and news in relation to the service. For example it will detail the last ~10 times the vacancies were updated and also people who requested for the service to call them all in chronological order. There will be a button will allows the service to view the full history. 
C. will be a graph of the history of vacancies over the past year. A line graph will probably be the best.
D.  will be a sort of sticky notes, the user can add in custom notes and remainders of their own. (Maybe even set up notifications they like). Case notes as well potentially.
E. will display information and stats of stuff beyond the service provider, such as information about how much people have used the service in Perth and how much vacancies overall. This will allow all services to have an idea of how well the system is working and maybe encourage changes if vacancies aren't readily available so there is a sense of responsibility overall rather than each service existing as just individuals cause to help homeless youth every service needs to help.
